rpcs3/util/v128.hpp

```diff
@@ -65,6 +65,9 @@ union alignas(16) v128
 
 	constexpr v128(const v128&) noexcept = default;
 
+	constexpr v128(const normal_array_t<u8>& _u8) noexcept : _u8(_u8){}
```

Comment:
You can use a template here, and do std::bit_cast<v128> for all masked_array_t

Reply:
Bit casting to union types isn't constexpr. But changing the existing constructor to this would work:
``` c++
template <Vector128 T>
constexpr v128(const T& rhs) noexcept
	: _u(std::bit_cast<u128>(rhs))
{
}
```
However, it would technically be UB to access the vector through any member other than _u afterwards.
Or do you mean something different?

Reply:
I rather it not be constexpr than, for constexpr stuff you can do the trick with masked_array_t as you have done earlier.